package/resources/data/uom_seed.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="uom_category_bahmni" model="uom.category">
+        <field name="name">Bahmni</field>
+    </record>
+    <record id="uom_uom_1" model="uom.uom">
+        <field name="name">Unit(s)</field>
+        <field name="category_id" ref="uom_category_bahmni"/>
+        <field name="factor">1.0</field>
+        <field name="rounding">0.01</field>
+        <field name="uom_type">reference</field>
+        <field name="active" eval="True"/>
+    </record>
+    <record id="uom_uom_2" model="uom.uom">
+        <field name="name">Tablet(s)</field>
+        <field name="category_id" ref="uom_category_bahmni"/>
+        <field name="factor">1.0</field>
+        <field name="rounding">0.01</field>
+        <field name="uom_type">bigger</field>
+        <field name="active" eval="True"/>
+    </record>
+    <record id="uom_uom_3" model="uom.uom">
+        <field name="name">Capsule(s)</field>
+        <field name="category_id" ref="uom_category_bahmni"/>
+        <field name="factor">1.0</field>
+        <field name="rounding">0.01</field>
+        <field name="uom_type">bigger</field>
+        <field name="active" eval="True"/>
+    </record>
+    <record id="uom_uom_4" model="uom.uom">
+        <field name="name">International Unit</field>
```

Comment:
Why is the need for this ?

Reply:
In pharmacology, the international unit (IU) is a unit of measurement for the effect or biological activity of a substance, for the purpose of easier comparison across similar forms of substances. Akhil suggested that we add this, for reference.